tests/integration_tests/with_db/udfs/test_prediction_with_downloader_udf.py

```diff
@@ -0,0 +1,63 @@
+import time
+from tests.utils import postprocessing
+
+SUB_DIR = 'test_downloader_with_prediction_sub_dir'
+MODEL_NAME = 'gaunernst/bert-tiny-uncased'
+
+
+def test_prediction_with_downloader_udf(
+        setup_database, pyexasol_connection, bucketfs_location):
+    bucketfs_conn_name, schema_name = setup_database
+
+    try:
+        # execute downloader UDF
+        input_data = (
+            MODEL_NAME,
+            SUB_DIR,
+            bucketfs_conn_name,
+            ''
+        )
+        query = f"""
+            SELECT TE_MODEL_DOWNLOADER_UDF(
+            t.model_name,
+            t.sub_dir,
+            t.bucketfs_conn_name,
+            t.token_conn_name
+            ) FROM (VALUES {str(input_data)} AS
```

Comment:
```suggestion
            ) FROM (VALUES {input_data} AS
```
`f"{val}"` will call `str(val)/__str__` implicitly on val

Reply:
True - copy/paste without thinking. Never mind, it's like that in many other places in the test code. Will clean it all at once. I will create a ticket for that.